pkg/controllers/v1alpha1/databackup/implement.go

```diff
@@ -304,6 +304,16 @@ func (r *DataBackupReconcilerImplement) generateDataBackupValueFile(ctx reconcil
 	nodeName, ip, rpcPort := utils.GetAddressOfMaster(masterPod)
 
 	imageName, imageTag := docker.GetWorkerImage(r.Client, databackup.Spec.Dataset, "alluxio", databackup.Namespace)
+	databackupImage := common.DEFAULT_ALLUXIO_RUNTIME_IMAGE
+	databackupImageInfo := strings.Split(databackupImage, ":")
+	if len(imageName) == 0 {
+		defaultImageName := databackupImageInfo[0]
```

Comment:
make sure the length of `databackupImageInfo` is greater than or equal to 1, Otherwise result in a panic

Reply:
Yes, if length of databackupImageInfo is 0, it will panic because index out of range